aws-attach-roles/iam-kubecost-athena-policy.json

```diff
@@ -20,33 +20,49 @@
         "arn:aws:glue:*:*:table/athenacurcfn*/*"
       ]
     },
+    {
+      "Sid": "AthenaQueryBucketAccess",
+      "Effect": "Allow",
+      "Action": [
+        "s3:GetBucketLocation",
+        "s3:ListBucket"
+      ],
+      "Resource": [
+        "arn:aws:s3:::aws-athena-query-results-bucket"
```

Comment:
Add comment here as a note for me

We also need to add this as additional resources here:
`"arn:aws:s3:::aws-athena-query-results-bucket/*"`